src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+
+	private enum Action{
+		ADD, REMOVE;
+	}
+	
+	private static final String ADD_ICON = "+";
+	private static final String REMOVE_ICON = "-";
+	
+	private Name personName;
+	private Tag tag;
+	private String icon;
+	
+	public Tagging(Action action, Tag tag, Name personName){
+		this.tag = tag;
+		this.personName = personName;
+		
+		if(action == Action.ADD){
+			icon = ADD_ICON;
```

Comment:
the Enum Action was already given inside a parameter, would storing the action as a String be better than the Enum itself?
The toString element can be used instead to handle the Action instead